### ISSUE TITLE
Would it be possible to add obstacles in the scene?

### ISSUE THREAD

**Issue by Morphlng** [jjshoots/PyFlyt#60]
Hi! This is a fantastic work for UAV simulation and RL learning. It would be nice if we can add some static and dynamic obstacles to increase the difficulty of task. I'm wondering if this is on the roadmap?

**Comment by jet-sony** [comment]
Yes, definitely on the roadmap and I've also received several emails requesting this. The issue ATM is I don't really have the bandwidth to add something like that yet. I would imagine some additions to the action space needs to happen (LiDar sensors(?), depth maps, etc).

If anyone is willing to implement that and make a PR, I'm more than happy to pick it up, clean it up, and implement the docs for it.

My current two focuses in terms of this library are in the remaining 2 PRs.

**Comment by evanofficial** [comment]
took a stab at the static side of this — basic URDF obstacles wired into the aviary with a couple primitive shapes shipped in. opening a PR now, figured starting small was the safer way in before tackling lidar/depth which feel like their own thing
